data/ui/book_detail.blp

```diff
@@ -73,24 +74,27 @@ template $BookDetail: Adw.NavigationPage {
                     ]
                   }
 
+                  Image fallback_icon {
+                    pixel-size: 200;
+                  }
+
                   styles [
                     "card",
                   ]
                 }
 
                 Box {
                   orientation: vertical;
-                  margin-top: 12;
-                  margin-bottom: 12;
                   spacing: 6;
                   vexpand: false;
 
+
                   Label book_label {
                     halign: start;
                     label: bind template.book_title;
                     ellipsize: end;
                     max-width-chars: 35;
-                    lines: 1;
+                    lines: 2;
```

Comment:
This is indeed better, and fortunately doesn't affect anything when there's only one line.